### PR TITLE
16.0 fix hr expense fix price praw

### PR DESCRIPTION
**Pull request by praw-odoo**
1)
Before this Commit, While adding journal lines in hr expense the amount per unit
was calculated as the total amount of product such as quantity into the price per
unit

After this Commit, the price per unit was calculated correctly

2)
Before this Commit, project > reporting > tasks analysis > drill down the
graph/pivot view > list view was editable and able to creatable

After this Commit,  project > reporting > tasks analysis > drill down the
graph/pivot view > list view was not editable and able to creatable

3)
Before this commit, in the project, the recurrent boolean was shown twice

After this commit, in the project, the recurrent boolean will show once only

task-3000758

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_expense/models/hr_expense.py [hunk 1]
```diff
@@ -602,7 +602,7 @@ def action_move_create(self):
                     Command.create({
                         'name': expense.employee_id.name + ': ' + expense.name.split('\n')[0][:64],
                         'quantity': expense.quantity or 1,
-                        'price_unit': expense.total_amount,
+                        'price_unit': expense.unit_amount if expense.unit_amount != 0 else expense.total_amount,
```
Comment (IT-Ideas) [inline addons/hr_expense/models/hr_expense.py:605]: Thanks for the fix 🙂 
Did you check that previous versions were not affected?
Thanks!

File: addons/project/views/project_views.xml [hunk 2]
```diff
@@ -1171,7 +1171,6 @@
                             <field name="partner_id" widget="res_partner_many2one" class="o_task_customer_field"/>
                             <field name="partner_phone" widget="phone" attrs="{'invisible': True}"/>
                             <field name="date_deadline" attrs="{'invisible': [('is_closed', '=', True)]}"/>
-                            <field name="recurring_task" attrs="{'invisible': ['|', ('allow_recurring_tasks', '=', False), ('active', '=', False)]}" groups="project.group_project_recurring_tasks"/>
```
Comment (IT-Ideas) [inline addons/project/views/project_views.xml:1174]: Could you rephrase your commit as:
```txt
[FIX] project: remove duplicate recurring_task field from task form view

task-3000758
```

File: addons/hr_expense/models/hr_expense.py (hunk 1 above)
Comment (mba-odoo) [reply addons/hr_expense/models/hr_expense.py:605]: @IT-Ideas , No, it's not affected in the previous version. Thanks

**Comment by mba-odoo** [comment]
@IT-Ideas , Done all the changes. except for the last improvement commit,  no other issue was present in the previous version. Thanks

**Comment by IT-Ideas** [comment]
Thanks 🙂 
@robodoo r+

**Comment by IT-Ideas** [comment]
@robodoo rebase-ff
